lawnchair/src/app/lawnchair/search/algorithms/engine/provider/apps/AppSearchProvider.kt

```
@@ -20,15 +29,65 @@ object AppSearchProvider {
+            )
+
+            return appResults.map { (appInfo, score) ->
+                val packageName = appInfo.targetComponent.packageName
+
+                val scoreExp = exp(2*(score-1.0f))
+
+                var weight: Float
+
+                try {
+                    weight = weights.getValue(packageName ?: "").toFloat()
+                } catch (e: NoSuchElementException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)